os/include/tinyara/input/touchscreen.h

```diff
@@ -154,10 +157,27 @@ struct touch_point_s {
  * a touch from first contact until the end of the contact.
  */
 
+typedef CODE void (*touch_handler_t)(FAR void *arg);
```

Comment:
Same as other new definitions and structure.

Reply:
could you check my comment above?

Reply:
we took reference from audio: https://github.com/Samsung/TizenRT/blob/4c0da9d6c8d34964794f54a033f81cbd476327c8/os/include/tinyara/audio/ndp120.h#L64

We have used it in similar way as used in **ndp_handler_t**. 
I didn't understood what need to change.

Reply:
```
Let's add an empty line to distinguish context from above.
And please add some description for this. What does this mean?
```

Reply:
I mean description of new definitions and new structures

Reply:
I have added the description.